plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -140,6 +140,20 @@ public void testCreateWithInvalidPropertyFails()
                 .hasMessage("Catalog 'iceberg' does not support materialized view property 'invalid_property'");
     }
 
+    @Test
+    public void testCreateWithDuplicateSourceTableSucceeds()
+    {
+        assertUpdate("" +
+                "CREATE MATERIALIZED VIEW materialized_view_with_duplicate_source AS " +
+                "SELECT _bigint, _date FROM base_table1 " +
+                "UNION ALL " +
+                "SELECT _bigint, _date FROM base_table1 ");
+
+        assertUpdate("REFRESH MATERIALIZED VIEW materialized_view_with_duplicate_source");
```

Comment:
Does this assertion pass without the `count = 12` parameter ?

Reply:
I ran the test locally, assertion passed without count = 12, yes. 

Same behaviour in the test as in in testShowTables or testSessionCatalogSchema. I am not sure if iceberg plugin is specified to return the true count of rows updated / written

Reply:
That's strange, when I try to run it locally I get
```
java.lang.AssertionError: expected no update count, but got 12

	at org.testng.Assert.fail(Assert.java:94)
	at io.trino.testing.QueryAssertions.assertUpdate(QueryAssertions.java:88)
	at io.trino.testing.AbstractTestQueryFramework.assertUpdate(AbstractTestQueryFramework.java:235)
	at io.trino.testing.AbstractTestQueryFramework.assertUpdate(AbstractTestQueryFramework.java:230)
	at io.trino.plugin.iceberg.TestIcebergMaterializedViews.testCreateWithDuplicateSourceTableSucceeds(TestIcebergMaterializedViews.java:474)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
```
All the other assertions in this file with `REFRESH MATERIALIZED VIEW` include the count parameter as well

Reply:
again you are right, apologies. Do not know where I was looking and the test now failed without the parameter locally. So this has been added. sorry for the mess up.